### DISCUSSION TITLE
AutoFill Credit policy

### DISCUSSION THREAD

**Discussion by kdc415** [mozilla/policy-templates#799] [Ideas]
It sure would be nice to block domain users from storing credit card info.

**Comment by mkaply** [comment]
You can set the preferences 

extensions.formautofill.creditCards.enabled
extensions.formautofill.creditCards.available 

to false 

But you're right, we should have policies for them.

I already have a bug for this, - https://bugzilla.mozilla.org/show_bug.cgi?id=1677524 - just haven't had a chance.

**Comment by mkaply** [comment]
Policies were added.
